ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -26,6 +28,12 @@ export default function FeeCard({
     bestQuoteText = t('swapBetterQuoteAvailable');
   }
 
+  const getTranslatedNetworkName = () => {
+    const networkNameKey =
+      SWAPS_CHAIN_ID_TO_NETWORK_NAME_KEY_MAP[chainId] || 'networkNameEthereum';
```

Comment:
Maybe we should throw here instead of defaulting to `'networkNameEthereum'`? :thinking: This should be unreachable right now, and in the future could only be hit if we add support for a fourth network in swaps but forget to add a network name. That scenario would be worth throwing an error for.

Unless there's some case I'm missing.

Reply:
It sounds fine to me. What about something like this? 
```
if (!networkNameKey) {
  throw new Error('This network is not supported for token swaps');
}
```

Reply:
That works!